CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### Changed
 
+- Adjusted the Firecracker [tenets](CHARTER.md) based on what we have learned
+  over the past 2 years.
```

Comment:
I would say this is not needed. In the changelog we keep track of Firecracker changes, so that a customer using version X could quickly evaluate what is changed in X+1 or X+2.
The tenet change isn't related to any version.

Reply:
Makes sense, will remove.